test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt

```diff
@@ -903,29 +991,63 @@ class QueryTests {
     }
 
     @Test
-    @Ignore
     fun sum_asFlow() {
-        // TODO
+        for (propertyDescriptor in allPropertyDescriptorsForSum) {
+            aggregatorAssertions(AggregatorQueryType.SUM, propertyDescriptor)
+        }
     }
 
     @Test
-    @Ignore
     fun sum_asFlow_throwsIfInvalidType() {
-        // TODO
+        val value1 = 2
+        val value2 = 7
+
+        realm.writeBlocking {
+            copyToRealm(QuerySample(intField = value1))
+            copyToRealm(QuerySample(intField = value2))
+        }
+
+        runBlocking {
+            assertFailsWith<IllegalArgumentException> {
+                realm.query<QuerySample>()
+                    .sum<String>(QuerySample::intField.name)
+                    .asFlow()
+                    .collect { /* No-op */ }
+            }.let {
+                assertTrue(it.message!!.contains("Invalid numeric type"))
+            }
+        }
     }
 
     @Test
-    @Ignore
     fun sum_asFlow_throwsIfInvalidProperty() {
-        // TODO
+        runBlocking {
+            assertFailsWith<IllegalArgumentException> {
+                realm.query<QuerySample>()
+                    .sum<String>(QuerySample::stringField.name)
+                    .asFlow()
+                    .collect { /* No-op */ }
+            }.let {
+                assertTrue(it.message!!.contains("Invalid query formulation"))
```

Comment:
Again, is the error test explanatory enough for people to understand it. Or is there more context in the full message?

Reply:
"Invalid query formulation" is thrown when we receive a `RealmCoreLogicException` or we are processing `genericRealmCoreExceptionHandler`. The message is delivered in full like so: `"Invalid query formulation: ${exception.message}"` being `exception.message` the actual message from core, which in most cases (as far as I've tested) should be clear enough.